tools/rna_tools/sshmm/sshmm.xml

```diff
@@ -0,0 +1,234 @@
+              from_work_dir="prepro_out/shapes/prepro_id/positive.txt">
+        <filter>
+            output_options['output_prepro_files'] is True and str_pred_method == 'rnashapes'
+        </filter>
+        </data>
+        <data format="txt" name="positive_rnastructures_out_file" 
+              label="${tool.name} on ${on_string} RNAstructures structures for sequences (txt)" 
+              from_work_dir="prepro_out/structure/prepro_id/positive.txt">
+        <filter>
+            output_options['output_prepro_files'] is True and str_pred_method == 'rnastructures'
+        </filter>
+        </data>
+        <data format="png" name="final_graph_png" 
+              label="${tool.name} on ${on_string} final graph (png)"
+              from_work_dir="results/res_out/final_graph.png"/>
+        <data format="png" name="logo_best_sequences_png" 
+              label="${tool.name} on ${on_string} logo best sequences (png)"
+              from_work_dir="results/res_out/logo_best_sequences.png">
+        <filter>
+            output_options['output_logo_files'] is True
+        </filter>
+        </data>
+        <data format="png" name="logo_global_png" 
+              label="${tool.name} on ${on_string} logo global (png)"
+              from_work_dir="results/res_out/logo_global.png">
+        <filter>
+            output_options['output_logo_files'] is True
+        </filter>
+        </data>
+        <data format="png" name="logo_hairpin_png" 
+              label="${tool.name} on ${on_string} logo hairpin (png)"
+              from_work_dir="results/res_out/logo_hairpin.png">
+        <filter>
+            output_options['output_logo_files'] is True
+        </filter>
+        </data>
+        <data format="txt" name="logo_best_sequences_txt" 
+              label="${tool.name} on ${on_string} logo best sequences (txt)"
+              from_work_dir="results/res_out/logo_best_sequences.txt">
+        <filter>
+            output_options['output_raw_files'] is True
+        </filter>
+        </data>
+        <data format="txt" name="logo_global_txt" 
+              label="${tool.name} on ${on_string} logo global (txt)"
+              from_work_dir="results/res_out/logo_global.txt">
+        <filter>
+            output_options['output_raw_files'] is True
+        </filter>
+        </data>
+        <data format="txt" name="logo_hairpin_txt" 
+              label="${tool.name} on ${on_string} logo hairpin (txt)"
+              from_work_dir="results/res_out/logo_hairpin.txt">
+        <filter>
+            output_options['output_raw_files'] is True
+        </filter>
+        </data>
+
+        <data format="xml" name="final_model_xml" 
+              label="${tool.name} on ${on_string} final model (xml)"
+              from_work_dir="results/res_out/final_model.xml">
+        <filter>
+            output_options['output_raw_files'] is True
+        </filter>
+        </data>
+    </outputs>
+    <tests>
+        <test>
```

Comment:
Can you use the num outputs as well in tests?

Reply:
The txt outputs change completely since the motifs change between every run (at least for the small test example). I had them in at the beginning, but comparing them doesn't make sense. Maybe some "file exists" check?

Reply:
Ich meine https://docs.galaxyproject.org/en/latest/dev/schema.html#id76 `expect_num_outputs` wie viele outputs erwartest du in dem test....

Reply:
Okay. Gibt's denn nen Test der einfach schaut ob ein file vorhanden ist?